### ISSUE TITLE
[FEATURE] Add `soci-snapshotter-grpc config` command

### ISSUE THREAD

**Issue by Kern--** [awslabs/soci-snapshotter#568]
### Description

Containerd has a [config command](https://github.com/containerd/containerd/blob/main/cmd/containerd/command/config.go) that can dump either the default config or the actual config as parsed from the config files. We should consider a similar command.

### Describe the solution you'd like

I think this is open for suggestion, but I would like the commands to only dump the actual configs with a not about where it's getting it's data. E.g.

```
$ soci-snapshotter-grpc config
# config based on /etc/soci-snapshotter-grpc/config.toml
metrics_address = "localhost:1339"
...
```
or if there is no config file
```
$ soci-snapshotter-grpc config
# config based on the default config
metrics_address = ""
...
```

The important piece is that the output should be valid toml and nothing else so that you can just write it to a file. E.g.
```
$ soci-snapshotter-gprc config > /etc/soci-snapshotter-grpc/config.toml
```

Should write out a config to `/etc/soci-snapshotter-grpc/config.toml` with all of the non-nullable config parameters set.

**Comment by sparr** [comment]
This would include every config parameter, even the ones currently set to defaults (which I don't think we can currently entirely distinguish)? So loading this exported config would reproduce the current exact config even if the defaults change later?

**Comment by Kern--** [comment]
Yes, I think that's the correct behavior. If you don't want fixed values, then you wouldn't overwrite your existing config file.

Effectively, it's a mechanism to explicitly set the defaults so that they don't change.
